### PR TITLE
[ZT] add WARP modes page

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

File: content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-modes.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+This mode is best suited for organizations that want to filter traffic directed to specific applications.
+
+| DNS filtering | HTTP filtering | Features enabled |
+| ------------- | -------------- | ---------------- |
+| No | Yes | HTTP policies, Browser Isolation, identity-based policies, AV scanning, and Data Loss Prevention for traffic sent through localhost proxy|
+
+{{<Aside type="note">}}
+
+ * Proxy mode can only be used by applications/operating systems that support SOCKS5/HTTPS proxy communication.
+ * This mode is only available on Windows, Linux and macOS.
```
Comment (dcpena) [inline content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-modes.md]: Tiny nit, but our style guide uses the Oxford comma.

File: content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-modes.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+### Device Information Only
+
+This mode is best suited for organizations that only want to enforce device posture for Access applications. DNS, Network and HTTP traffic is handled by the default mechanisms on your devices.
+
+| DNS filtering | HTTP filtering | Features enabled |
+| ------------- | -------------- | ---------------- |
+| No | No | Device posture rules in [Access policies](/cloudflare-one/policies/access/) |
+
+When you enroll a device in Device Information Only mode, the WARP client will automatically create and install a client certificate on the device (unless you have previously installed the [Cloudflare certificate](/cloudflare-one/connections/connect-devices/warp/user-side-certificates/install-cloudflare-cert/)). The client certificate is necessary to confirm the source of outgoing traffic. You can view the certificate on the [Cloudflare dashboard](https://dash.cloudflare.com/) by selecting your domain and going to **SSL/TLS** > **Client Certificates**.
```
Comment (dcpena) [inline content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-modes.md:51]: Should Device Information Only be bold?

File: content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings.md [hunk 3]
```diff
@@ -201,11 +201,7 @@ When `Enabled`, the **Send Feedback** button in the WARP client appears and will
 </div>
 </details>
 
-Allows you to choose the operational mode of the client. Refer to [WARP Modes](/cloudflare-one/connections/connect-devices/warp/#warp-client-modes) for a detailed description of each mode.
-
-- **Gateway with WARP** DNS and Device traffic is encrypted and processed by Gateway. This mode is required if you want to enable HTTP rules, Browser Isolation, Anti-Virus scanning and DLP.
-- **Gateway with DoH** Enforcement of DNS policies only through DoH. All other traffic is handled by default mechanisms on your devices.
-- **Proxy Only** Only traffic sent to the localhost proxy is encrypted by Gateway. This mode does not process DNS traffic.
+Allows you to choose the operational mode of the client. Refer to [WARP Modes](/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-modes) for a detailed description of each mode.
```
Comment (dcpena) [inline content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings.md:204]: This looks good, but JFYI, line 189 is missing a closing tick mark in "Use a mailto: link to open your..." which is what's causing the text to look blue.

**Review by dcpena — APPROVED** [review]

File: content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-modes.md (hunk 1 above)
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-modes.md]:
```suggestion
 * This mode is only available on Windows, Linux, and macOS.
```

**Pull request merged**